public/scss/pages/PlaceorderV2.module.scss

```
@@ -0,0 +1,487 @@
+        margin: 0;
+        padding: 0;
+      }
+
+      &:hover,
+      &:focus-visible
+      {
+        background-color: rgba(247, 190, 22, 0.1);
+      }
+    }
+  }
+}
+
+.mapSearchBar
+{
+  @include flex(row, center, flex-start);
+  @include fixedWidth(calc(100% - (52px + 20px)));
+  position: relative;
+  margin: 0;
+  min-height: 52px;
+
+  & > span
+  {
+    display: none;
+  }
+}
+
+.mapSearchBarInput
+{
+  @include transition;
+  @include fixedHeight(52px);
+  @include typographyBuilder(400, 16, 24);
+  flex-grow: 1;
+  margin: 0;
+  padding: 0 20px 0 calc((16px * 2) + 8px) !important;
+
+  border: solid 1px $color_black;
+  box-shadow: none;
+  background-color: $color_white;
+  background-repeat: no-repeat;
+  background-position: 16px center;
+  background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' viewBox='0 0 16 16' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M12.0208 11.0779L14.8762 13.9326L13.9328 14.8759L11.0782 12.0206C10.016 12.8721 8.69483 13.3352 7.3335 13.3332C4.0215 13.3332 1.3335 10.6452 1.3335 7.33325C1.3335 4.02125 4.0215 1.33325 7.3335 1.33325C10.6455 1.33325 13.3335 4.02125 13.3335 7.33325C13.3354 8.69458 12.8723 10.0157 12.0208 11.0779ZM10.6835 10.5832C11.5296 9.71317 12.0021 8.54687 12.0002 7.33325C12.0002 4.75458 9.9115 2.66658 7.3335 2.66658C4.75483 2.66658 2.66683 4.75458 2.66683 7.33325C2.66683 9.91125 4.75483 11.9999 7.3335 11.9999C8.54712 12.0018 9.71342 11.5293 10.5835 10.6832L10.6835 10.5832Z' fill='%232A2A2A'/%3E%3C/svg%3E%0A");
+
+  &:hover,
+  &:focus,
+  &:focus-visible,
+  &:active
+  {
+    outline: none;
+    border: solid 1px $color_black;
+    box-shadow: none;
+  }
+
+}
+
+.mapCenterButtonContainer
+{
+  position: absolute;
+  top: calc(100% + 12px);
+  right: 20px;
+  z-index: 2;
+}
+
+.mapCenterIcon
+{
+  color: $color_text_secondary;
+  height: 20px;
+  width: 20px;
+}
+
+.mapPopupCenterButton
+{
+  @include transition;
+  @include flex(row, center, center);
+  @include typographyBuilder(400, 16, 24);
+  position: relative;
+  margin: 0;
+  padding: 0 10px;
+  border: none;
+  box-shadow: none;
+  background-color: $color_white;
+  color: $color_black;
+  min-height: 40px;
+  box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.2);
+
+  &:hover,
+  &:focus-visible
+  {
+    background-color: $color_grey;
+  }
+}
+
+.mapFormAddress
+{
+  position: relative;
+  padding: 16px 20px 0;
+}
+
+.mapFooterContainer
+{
+  margin-top: 72px;
+
+  @media screen and (max-width: 479px)
+  {
+    margin-top: 0;
+    padding-top: 72px;
+  }
+
+  -moz-box-flex: 1;
+  flex-grow: 1;
+  overflow: auto;
+}
+
+.mapButtonFooterContainer
+{
+  width: 100%;
+  position: relative;
+}
+
+.mapButtonFooter
+{
+  @extend .btn_primary;
+  width: 100%;
+  height: 52px;
+  background-color: $color_black !important;
+  
+  &:hover
+  {
+    background-color: $color_black !important;
+  }
+}
+
+.mapDistrictLabel
+{
+  @include fontPrimary(400);
+  @include fontSize(16);
+  @include lineHeight(24);
```

```suggestion
  @include typographyBuilder(400, 16, 24);
```

pake mixin ini bisa sepertinya